<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>WAVFormatChooserWidget</class>
    <widget class="QWidget" name="WAVFormatChooserWidget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>74</height>
            </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
                <number>4</number>
            </property>
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_1">
                    <item>
                        <widget class="QLabel" name="label_8">
                            <property name="text">
                                <string>Sample rate</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QComboBox" name="sampleRateComboBox">
                            <property name="currentIndex">
                                <number>0</number>
                            </property>
                            <item>
                                <property name="text">
                                    <string>44100</string>
                                </property>
                            </item>
                            <item>
                                <property name="text">
                                    <string>48000</string>
                                </property>
                            </item>
                            <item>
                                <property name="text">
                                    <string>96000</string>
                                </property>
                            </item>
                            <item>
                                <property name="text">
                                    <string>11025</string>
                                </property>
                            </item>
                            <item>
                                <property name="text">
                                    <string>22050</string>
                                </property>
                            </item>
                            <item>
                                <property name="text">
                                    <string>88200</string>
                                </property>
                            </item>
                            <item>
                                <property name="text">
                                    <string>192000</string>
                                </property>
                            </item>
                            <item>
                                <property name="text">
                                    <string>384000</string>
                                </property>
                            </item>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Orientation::Horizontal</enum>
                            </property>
                            <property name="sizeType">
                                <enum>QSizePolicy::Policy::Fixed</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>8</width>
                                    <height>1</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QLabel" name="label_10">
                            <property name="text">
                                <string>Bit depth</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QComboBox" name="sampleTypeComboBox"/>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer_1">
                            <property name="orientation">
                                <enum>Qt::Orientation::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                        <widget class="QLabel" name="label_9">
                            <property name="text">
                                <string>Channels</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QComboBox" name="channelsComboBox"/>
                    </item>
                    <item>
                        <widget class="QSpinBox" name="channelsSpinBox">
                            <property name="minimum">
                                <number>1</number>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer_3">
                            <property name="orientation">
                                <enum>Qt::Orientation::Horizontal</enum>
                            </property>
                            <property name="sizeType">
                                <enum>QSizePolicy::Policy::Fixed</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>8</width>
                                    <height>1</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QLabel" name="label_11">
                            <property name="text">
                                <string>Container</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QComboBox" name="containerFormatComboBox"/>
                    </item>
                    <item>
                        <widget class="QToolButton" name="formatHelpButton">
                            <property name="toolTip">
                                <string>Show WAV container format information</string>
                            </property>
                            <property name="text">
                                <string>?</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer_2">
                            <property name="orientation">
                                <enum>Qt::Orientation::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections>
        <connection>
            <sender>formatHelpButton</sender>
            <signal>clicked()</signal>
            <receiver>WAVFormatChooserWidget</receiver>
            <slot>showFormatHelp()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>595</x>
                    <y>21</y>
                </hint>
                <hint type="destinationlabel">
                    <x>378</x>
                    <y>16</y>
                </hint>
            </hints>
        </connection>
    </connections>
    <slots>
        <slot>showFormatHelp()</slot>
    </slots>
</ui>
